src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -785,23 +787,27 @@ namespace OpenLoco::Audio
         }
     }
 
-    static void off_4FEB58(Vehicles::VehicleSoundPlayer* v, int32_t x)
+    static void processVehicleForSound(Vehicles::VehicleSoundPlayer* v, int32_t step)
     {
-        switch (x)
+        switch (step)
         {
             case 0:
+                // 0x048A262
                 v->soundFlags &= ~Vehicles::SoundFlags::flag0;
                 break;
             case 1:
+                // TODO: this does not match IDA
+                // 0x048A268
```

Comment:
@duncanspumpkin Case 1 and 2 were identical, so I got curious. Case 1 doesn't seem to match disasm in IDA, though. Do you perhaps remember why?